deployment/helpers.go

```diff
@@ -87,7 +88,7 @@ func parseError(txError error) (string, error) {
 	return callErr.Data, nil
 }
 
-func ParseErrorFromABI(errorString string, contractABI string) (string, error) {
+func parseErrorFromABI(errorString string, contractABI string) (string, error) {
```

Comment:
i didn't find any uses of this in [cld](https://github.com/search?q=repo%3Asmartcontractkit%2Fchainlink-deployments%20ParseErrorFromABI&type=code)